lib/client.js

```diff
@@ -126,15 +126,16 @@ exports.getHistory = async ({
   return res.data;
 };
 
-exports.notifyPeer = async ({localPeerId, remotePeer}) => {
+exports.notifyPeer = async ({localPeerId, remotePeer, ledgerNodeId}) => {
```

Comment:
It seems to me that all of this must be driven off of the `ledgerNodeId`, so we shouldn't pass the `localPeerId` here. It might even be in conflict with whatever is retrieved via `_getKeyPair`. It looks like `_getKeyPair` returns the `localPeerId` anyway (and it's guaranteed to match): https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/blob/7c1fd0d1925a8caae15c8a0e7e2ab0ec52de4c4e/lib/localPeers.js#L271

```suggestion
exports.notifyPeer = async ({ledgerNodeId, remotePeer}) => {
```